rts/Sim/Units/CommandAI/MobileCAI.cpp

```diff
@@ -74,7 +74,6 @@ CR_REG_METADATA(CMobileCAI, (
 	CR_MEMBER(lastCommandFrame),
 	CR_MEMBER(lastCloseInTry),
 	CR_MEMBER(lastBuggerOffTime),
-	CR_MEMBER(numNonMovingCalls),
```

Comment:
should be a rename and not a deletion, I think?

Reply:
I agree

Reply:
Thanks for spotting that. I've corrected this in the mainline.  https://github.com/beyond-all-reason/spring/commit/852e710368be303ae816d322a54281745c9c4c76

Reply:
Sorry if that felt abrupt: we're going through a release cycle and I'm eager to see all issues pushed through and tested by the test team ASAP.